android/java/org/chromium/chrome/browser/settings/BraveSyncCodeCountdownFragment.java

```diff
@@ -75,6 +75,8 @@ public void run() {
                 1000);
     }
 
+    // duration.getSeconds should be toSeconds after api 31.
+    @SuppressWarnings("JavaDurationGetSecondsToToSeconds")
```

Comment:
why are we suppressing this warning instead of changing the code?

Reply:
cc: @AlexeyBarabash